(bug) The configuration for Google Cloud Storage Retriever is incorrect when read from environment variables

### Observed behavior

I plan to use **relay proxy** with Google Storage Retriever. I am providing environment variables as follows:
```
RETRIEVER_KIND=googleStorage
RETRIEVER_BUCKET=my-bucket
RETRIEVER_OBJECT=path/to/my-flags.yaml
```
But **relay proxy** cannot start up with error message:
```
panic: impossible to retrieve the flags, please check your configuration: storage: object doesn't exist
```

### Expected Behavior

I checked the bucket and the object path, both are correct and readable if I use the Go module. So **relay proxy** must also work.

### Steps to reproduce

Start **relay proxy** with the following environment variables:
```
RETRIEVER_KIND=googleStorage
RETRIEVER_BUCKET=my-bucket
RETRIEVER_OBJECT=path/to/my-flags.yaml
```

--- a/cmd/relayproxy/config/retriever.go
+++ b/cmd/relayproxy/config/retriever.go
@@ -16,7 +16,7 @@ type RetrieverConf struct {
 	HTTPBody    string              `mapstructure:"body" koanf:"body"`
 	HTTPHeaders map[string][]string `mapstructure:"headers" koanf:"headers"`
 	Bucket      string              `mapstructure:"bucket" koanf:"bucket"`
-	Object      string              `mapstructure:"bucket" koanf:"bucket"`
+	Object      string              `mapstructure:"object" koanf:"object"`
 	Item        string              `mapstructure:"item" koanf:"item"`
 	Namespace   string              `mapstructure:"namespace" koanf:"namespace"`
 	ConfigMap   string              `mapstructure:"configmap" koanf:"configmap"`

